internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_provider.go

```diff
@@ -205,12 +205,9 @@ func DataSource() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"audience_claim": {
-				Type:     schema.TypeList,
+			"audience": {
```

Comment:
is this a breaking change?

Reply:
yes it is, I added more entries to the changelog entry file so it's clear what we have changed. Let me know what you think

Reply:
that's fine thanks.  
as It's a breaking change, please make sure that a migration guide file is created with instructions.
also add ! to the PR like:   feat!: Adds create ...

https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/development-setup.md
```
a ! in the description of the PR: The commit introduces a breaking change (correlating with MAJOR in Semantic Versioning). A breaking change can be part of commits of any type. Examples:
fix!: Description of the ticket
```

https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github/pr-labeler.yml#L4

Reply:
Thanks Leo! I have added the upgrade guide and also changed the PR title to include `!`. I will wait to merge this PR after 1.16.2 has been released.

Reply:
knit: Type of change in PR description has to be changed to check  breaking change